src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,55 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './CurrentUser.scss';
+import { getUser } from '../../api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+interface Props {
+  selectedUserId: number,
+  onHandlerClear: (userId: number) => void,
+}
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = ({
+  selectedUserId,
+  onHandlerClear,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(selectedUserId)
+      .then(userFromServer => setUser(userFromServer));
```

Comment:
you have to handle errors, if getUser returns an error you should catch error here